Jenkinsfile

```diff
@@ -32,6 +32,8 @@ pipeline {
         LINUX_MVN_RANDOM = '-Djava.security.egd=file:/dev/./urandom'
         COVERAGE_EXCLUSIONS = '**/test/**/*,**/itests/**/*,**/*Test*,**/sdk/**/*,**/*.js,**/node_modules/**/*,**/jaxb/**/*,**/wsdl/**/*,**/nces/sws/**/*,**/*.adoc,**/*.txt,**/*.xml,/**/IT*.java'
         DISABLE_DOWNLOAD_PROGRESS_OPTS = '-Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn '
+        SOURCE_NODE = 'https://ditto-1.phx.connexta.com:8993/services'
```

Comment:
I don't think we want these properties in our jenkins file. We should be able to pass them in through the settings file or maybe even just the jenkins job itself.

Reply:
We might need to add a custom replication settings.xml that defines these properties and then include that settings in our withMaven call

Reply:
These should be defined on Jenkins and not checked in our repo at all. We should document that we require these properties to be defined. The goal is not to hardcode knowledge of the environment where we will be building.